SimPEG/Survey.py

```diff
@@ -149,8 +149,8 @@ def vnD(self):
         return np.array([rx.nD for rx in self.rxList])
 
 
-class Data(object):
-    """Fancy data storage by Src and Rx"""
+class BaseData(object):
```

Comment:
- [ ] @rowanc1: can you take a look at these changes?

I am fine with this - I think down the road, using properties here would be a further improvement, but this is a step up from the current status